common/beerocks/bcl/source/son/son_wireless_utils.cpp

```diff
@@ -454,4 +454,59 @@ std::vector<uint8_t> wireless_utils::calc_5g_20MHz_subband_channels(
                   [](uint8_t channnel) { LOG(DEBUG) << "channnel = " << int(channnel); });
 
     return channels;
-}
\ No newline at end of file
+}
+
+/**
+ * @brief convert operating class to channel set based on Table 4-E in the ieee 802.11 specification
+ *
+ * @param operating_class operating class
+ * @return std::unordered_set<uint8_t> set of supported channels by the operating class or empty if failure
+ */
+std::unordered_set<uint8_t> wireless_utils::operating_class_to_channel_set(uint8_t operating_class)
+{
+    static const std::map<uint8_t, std::unordered_set<uint8_t>> operating_class_to_channel_set = {
```

Comment:
I should have thought of this before, of course, but: an operating class is more than just a channel set, it also puts limits, including bandwidth. So maybe it would be more appropriate to use sWifiChannel directly?

But that can be done in a later commit.

Reply:
Part of the channel selection full flow then.